Pharmacy/Pharmacy/Model/Medicine.cs

```diff
@@ -1,8 +1,22 @@
-﻿namespace Pharmacy.Model
+﻿using System.Collections.Generic;
+
+namespace Pharmacy.Model
 {
     public class Medicine
     {
         public int Id { get; set; }
         public string Name { get; set; }
+        public int ManufacturerId { get; set; }
+        public Manufacturer Manufacturer { get; set; }
+        public List<string> SideEffects { get; set; }
```

Comment:
Would probably change these to IEnumerable<T> since they are more flexible (IEnumerable has an extension .ToList() if necessary, but interfaces give us more flexibility)